clients/cloud/groovy/src/main/groovy/io/confluent/examples/clients/cloud/util/PropertiesLoader.groovy

```diff
@@ -0,0 +1,20 @@
+package io.confluent.examples.clients.cloud.util
+
+import java.nio.file.Files
+import java.nio.file.Paths
+
+class PropertiesLoader {
+
+  static Properties loadConfig(String configFile) {
+    if (!Files.exists(Paths.get(configFile))) {
+      throw new IOException(configFile + " not found.")
+    }
+    def cfg = new Properties()
+
+    def propertiesFile = new File(configFile)
+    propertiesFile.withInputStream {
+      cfg.load(it)
```

Comment:
nit: using a named closure parameter seems a preferable style to me. i.e. `stream ->` rather than `it`. but i'm not a groovy person.

Reply:
You're generally right, but if the statement is clear and short, it actually help readability. What I read here is:
"This properties input stream, _config, load it!_". Perfect readbility to me.

Reply:
Actually `cfg.load it` would be even better.
I will fix

Reply:
done